### PR TITLE
Prerelease collector contrib v0.51.0

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@
...
+
+### 💡 Enhancements 💡
+
+### 🧰 Bug fixes 🧰
+
+## v0.50.0
+
+### 🛑 Breaking changes 🛑
+
 - `datadogexporter`: Replace HistogramMode defined as string with enum.
```
Comment (dmitryax) [inline CHANGELOG.md:19]:
```suggestion
- `datadogexporter`: Replace HistogramMode defined as string with enum (#9589)
```

Comment (dmitryax) [reply CHANGELOG.md:19]: And another one missed below

```
- `exporter/azuremonitor`: Deprecate use of LogRecord.Name as the log envelope category name. There is no replacement (#9258)
```

Comment (bogdandrutu) [reply CHANGELOG.md:19]: confused, that `azuremonitor` is already under deprecation?

Comment (bogdandrutu) [reply CHANGELOG.md:19]: Found it, you want to PR number, but that should be checked during review :)

Comment (bogdandrutu) [reply CHANGELOG.md]: Done.

**Review by dmitryax — APPROVED** [review]

**Pull request merged**